ci/cscs-ci.yml

```diff
@@ -156,7 +156,7 @@ build_py38_image_x86_64:
   variables:
     CRAY_CUDA_MPS: 1
     SLURM_JOB_NUM_NODES: 1
-    SLURM_TIMELIMIT: 15
+    SLURM_TIMELIMIT: 20
```

Comment:
@philip-paul-mueller The execution of gpu tests without optimization is probably very slow.